### PR TITLE
Click to recipes

### PR DESCRIPTION
**Pull request by DavidBronfen**
#32

### CODE REVIEW

**Comment by nirgn975** [comment]
@DavidBronfen The test failed because you forget to change the [initialState](https://github.com/DavidBronfen/Retrogusto/blob/master/client/src/app/reducers/categories.ts#L8) of the categories reducer.

**Comment by DavidBronfen** [comment]
@nirgn975 When clicking on the `rg-category` element it should console and alert some stuff, instead it throws an error:
developer_tools_-_http___localhost_4200_categories

Any idea what I did wrong?

**Comment by DavidBronfen** [comment]
https://github.com/angular/angular-cli/issues/4425

**Comment by DavidBronfen** [comment]
Last error was fixed by changing the `zone.js` version.
New error occurs:
job__185_2_-_davidbronfen_retrogusto_-_travis_ci

File: client/e2e/app.e2e-spec.ts [hunk 1]
```diff
@@ -6,4 +6,9 @@ describe('retrogusto App', () => {
   beforeEach(() => {
     page = new RetrogustoPage();
   });
+
```
Comment (nirgn975) [inline client/e2e/app.e2e-spec.ts]: This is why you have the error message. We don't yet to e2e tests, delete those new lines angular-cli did for you.

File: client/e2e/app.po.ts [hunk 2]
```diff
@@ -4,4 +4,8 @@ export class RetrogustoPage {
   navigateTo() {
     return browser.get('/');
   }
+
+  getParagraphText() {
```
Comment (nirgn975) [inline client/e2e/app.po.ts]: This is also related to e2e tests, delete those new lines too.

**Comment by DavidBronfen** [comment]
@nirgn975 It was an exhausting pull request  🎉

**Comment by DavidBronfen** [comment]
@nirgn975 What about the codecov? 
Should we merge it like this and in the master it will take effect of the 60% coverage we wanted it to be?

**Comment by nirgn975** [comment]
Yes we should, LGTM

**Pull request merged**